net-p2p/syncthing/syncthing-0.14.0-r1.ebuild

```diff
@@ -86,14 +86,11 @@ src_install() {
 	if use tools ; then
 		# openrc and systemd service files
 		systemd_dounit "${S}"/src/${EGO_PN}/cmd/strelaysrv/etc/linux-systemd/strelaysrv.service
-		newconfd "${FILESDIR}/${PN}-relaysrv.confd" ${PN}-relaysrv
-		newinitd "${FILESDIR}/${PN}-relaysrv.initd" ${PN}-relaysrv
-
-		keepdir /var/lib/${PN}-relaysrv
```

Comment:
Regards to @djc. These lines seem to have been wrong, and not needed. `/var/lib/strelaysrv` is a home dir for `strelaysrv` by design. We don't install any files to keep there. Correct ownership on `/var/log/syncthing/strelaysrv.log` is ensured by the openrc init script, while systemd does not use it.